build/teamcity/cockroach/nightlies/roachtest_nightly_impl.sh

```diff
@@ -67,13 +67,13 @@ build/teamcity-roachtest-invoke.sh \
   --cloud="${CLOUD}" \
   --count="${COUNT-1}" \
   --clear-cluster-cache="${CLEAR_CLUSTER_CACHE:-true}" \
-  --auto-kill-threshold="${AUTO_KILL_THRESHOLD:-0.05}" \
+  --auto-kill-threshold="${AUTO_KILL_THRESHOLD:-0.10}" \
   --parallelism="${PARALLELISM}" \
   --cpu-quota="${CPUQUOTA}" \
   --cluster-id="${TC_BUILD_ID}" \
   --artifacts=/artifacts \
   --artifacts-literal="${LITERAL_ARTIFACTS_DIR:-}" \
   --slack-token="${SLACK_TOKEN}" \
   --suite nightly \
-  --selective-tests="${SELECTIVE_TESTS:-false}" \
+  --selective-tests="${SELECTIVE_TESTS:-true}" \
```

Comment:
Since the default is now flipped, we can remove `env.SELECTIVE_TESTS` in CI, right?

Reply:
yes